cmd/utils/flags.go

```diff
@@ -1287,6 +1293,12 @@ func setLes(ctx *cli.Context, cfg *ethconfig.Config) {
 	if ctx.IsSet(LightNoSyncServeFlag.Name) {
 		cfg.LightNoSyncServe = ctx.Bool(LightNoSyncServeFlag.Name)
 	}
+	if ctx.IsSet(LightRecentStateFlag.Name) {
+		cfg.LightRecentState = ctx.Int(LightRecentStateFlag.Name)
+		if cfg.LightRecentState <= les.BlockSafetyMargin {
+			log.Error("LES recent state value is too small", "have", cfg.LightRecentState, "safetyMargin", les.BlockSafetyMargin)
```

Comment:
Instead of just printing an error, can we also update the value with `les.BlockSafetyMargin`?